typed/declaration/schema-schema.ipldsch.json

```diff
@@ -48,42 +67,30 @@
 				"map": null,
 				"list": null,
 				"link": null
+			},
+			"representation": {
+				"string": {}
 			}
 		},
 		"TypeBool": {
 			"kind": "struct",
-			"fields": {},
-			"representation": {
-				"map": {}
-			}
```

Comment:
For all these, whether or not we _want_ them here depends on [which of the two rules](https://github.com/ipld/go-ipld-prime/pull/10#issuecomment-488585486) we prefer.

But right _now_, the final word should match [the field defns for TypeStruct](https://github.com/ipld/go-ipld-prime/blob/f2d595797253f736988a9a2402c2a5a0a05d4e93/typed/declaration/schema-schema.ipldsch#L286) -- which have representation not being an optional field, so these should stay in by that logic.